v0/destinations/marketo/transform.js

```diff
@@ -70,11 +70,13 @@ const getAuthToken = async formattedDestination => {
     const data = marketoResponseHandler(
       clientResponse,
       "During fetching auth token",
-      TRANSFORMER_METRIC.TRANSFORMER_STAGE.TRANSFORM
+      TRANSFORMER_METRIC.TRANSFORMER_STAGE.TRANSFORM,
+      authCache,
+      formattedDestination.ID
```

Comment:
This is the token fetch request. It should not lead to cache eviction, as the token is definitely not present in the cache.